client/lib/media/utils.js

```diff
@@ -488,6 +488,8 @@ export function canUserDeleteItem( item, user, site ) {
  * @param {number} quality extracted image quality
  */
 export function canvasToBlob( canvas, callback, type, quality ) {
+	const { HTMLCanvasElement, Blob, atob } = window;
```

Comment:
this change definitely stands out. was this necessary or a separate fix? if we were to have to rollback the URL changes, would we also want this to rollback?

Reply:
This is due to linter config changes. It used to be ok to refer to these directly, but that's no longer allowed. I decided to destructure window here, instead of prepending `window.` to every usage of these in the method.